pint-types-bundle/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@pint/types",
+  "version": "1.0.0",
```

Comment:
`1.0.0`?  but we can override this field while publishing to the npm registry with command-line args in CI/scripts